src/core/src/pso/descriptor.rs

```diff
@@ -91,11 +91,11 @@ pub struct DescriptorSetWrite<'a, 'b, B: Backend> {
 #[allow(missing_docs)] //TODO
 pub enum DescriptorWrite<'a, B: Backend> {
     Sampler(Vec<&'a B::Sampler>),
-    SampledImage(Vec<(&'a B::ShaderResourceView, ImageLayout)>),
-    StorageImage(Vec<(&'a B::ShaderResourceView, ImageLayout)>),
-    UniformTexelBuffer,
-    StorageTexelBuffer,
-    ConstantBuffer(Vec<&'a B::ConstantBufferView>),
-    StorageBuffer,
-    InputAttachment(Vec<(&'a B::ShaderResourceView, ImageLayout)>),
+    SampledImage(Vec<(&'a B::ImageView, ImageLayout)>),
+    StorageImage(Vec<(&'a B::ImageView, ImageLayout)>),
+    UniformTexelBuffer(Vec<&'a B::BufferView>),
+    StorageTexelBuffer(Vec<&'a B::BufferView>),
+    ConstantBuffer(Vec<&'a B::BufferView>),
```

Comment:
Vulkan doesn't use a `BufferView` but a `Buffer` + `Range<u64>`.

Reply:
!!! Yeah, you are right. All buffer descriptors in Vulkan do not use `VkBufferView`. :/
Why does Vulkan provide `VkBufferView` handles?
[Spec](https://vulkan.lunarg.com/doc/view/1.0.37.0/linux/vkspec.chunked/ch11s02.html)

Reply:
I'm curious why `&Vec<&T>` instead of `&[&T]`?

Reply:
Fixed the `BufferView` here. Left `Vec` for the original code, was wondering about `&[]` alternative myself too. @msiglreith do you recall why you used `Vec` here?

Reply:
I think I had some builder pattern back then for descriptor updates. `&[]` should be the way to go.